docs/next.config.js

```diff
@@ -1,11 +1,11 @@
 const withNextra = require('nextra')({
   theme: 'nextra-theme-docs',
   themeConfig: './theme.config.tsx',
```

Comment:
The green is now super light and vibrant. Really hard to read on the light version:
After:
<img width="672" alt="Screenshot 2022-12-21 at 18 50 32" src="https://user-images.githubusercontent.com/5823445/208898722-7a54cf25-63d9-4c4b-a1c1-957a359383c0.png">
Before:
<img width="672" alt="Screenshot 2022-12-21 at 18 50 38" src="https://user-images.githubusercontent.com/5823445/208898744-11990cd4-0450-4b51-ab3a-8b2e9dbd13e1.png">

Reply:
color fixed, I had to replace the hack with a new hack